### PR TITLE
S2U-43 5.2.8.1 Admin: Ability to enrol, unenrol and disable multiple users in multiple sites - fix

### PR DESCRIPTION
**Pull request by JuanDavid102**

### CODE REVIEW

File: admin-bulk-user-membership/tool/src/main/java/org/sakaiproject/bulk/membership/controller/MainController.java [hunk 1]
```diff
@@ -206,14 +208,20 @@ public String verifyFistStep(@RequestParam(value="users", required = false) Stri
...
         int count;
         ArrayList<Summary> summaries = new ArrayList<Summary>();
+        int userCount = 1;
         for (String userCriteria : users) {
-            count = 0;
             Summary summary = new Summary(userCriteria);
+            User user = null;
+            try {
+                user = bulkUserMembershipToolService.getUser(userCriteria);
+            } catch (UsersByEmailException usEx) {}
```
Comment (jesusmmp) [inline admin-bulk-user-membership/tool/src/main/java/org/sakaiproject/bulk/membership/controller/MainController.java:217]: No message on the catch?

**Comment by mpellicer** [comment]
I don't get this, why are we doing special treatment over emails? usereids can be emails.

**Comment by jesusmmp** [comment]
Maybe should be easier to create a new method get on Summary which returns the first part of the id without @

image

**Comment by mpellicer** [comment]
Explain this to me, this is just a frontend issue resolved via backend, am I right? Am I able to use users that have an email as an EID?

**Comment by JuanDavid102** [comment]
Yes, It can be user the email and the EID to get the users. This is only because the data-parent can't resolve the '@', so we eliminate it with this function call

**Review by mpellicer — APPROVED** [review]

**Comment by mpellicer** [comment]
Nice thanks for the good fix @JuanDavid102

**Pull request merged**